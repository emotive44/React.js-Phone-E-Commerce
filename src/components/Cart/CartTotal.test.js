import React from 'react';
import CartTotal from './CartTotal';
import renderer from 'react-test-renderer';

jest.mock('react-router-dom', () => ({
    Link: 'Link'
}));

describe('CartTotal component', () => {
    test('should reder null if data is undefined', () => {
        const component = renderer.create(<CartTotal />);
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    })
})