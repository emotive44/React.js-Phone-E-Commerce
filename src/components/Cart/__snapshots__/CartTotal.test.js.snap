// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CartTotal component should reder null if data is undefined 1`] = `null`;
